js/modules/tamagotchi.js

```diff
@@ -37,4 +38,66 @@ export default class Tamagotchi {
     this.fun.element = funElement;
     this.displayFun(funElement);
   };
+
+  checkState() {
+    if (this.health.value <= 0) {
+      this.state = "dead";
```

Comment:
**nitpick:** Look how often you have to write the name of these states again, and again. What if the name of such state changes? You would have to find all occurrences in the app, and change them manually. It's very hard to maintain. Let's move these states to some `const`, like: 
```
this.states = {
    dead: "dead",
    hungry: "hungry",
    ...
}
```
Then you could just use it like: 
```
this.state = this.states.dead
```